x-pack/solutions/observability/plugins/streams/server/lib/streams/helpers/sync.ts

```diff
@@ -196,16 +197,15 @@ async function ensureStreamManagedPipelineReference(
         },
       },
     };
-    executionPlan.push({
```

Comment:
I replaced the executionPlan logic with elasticsearch calls since it does not seem to do much for now. Happy to revert if there are plans to add more to that construct